justfile

```diff
@@ -139,3 +139,6 @@ download-design-tokens:
 # Update the results of `insta` snapshot regression tests
 update-insta-tests:
     cargo test; cargo insta review
+
+upload *ARGS:
+    python3 "scripts/upload_image.py" {{ARGS}}
```

Comment:
please configure your editor to add newlines to end of files (and to trim trailing whitespace)!

Reply:
I do have it setup to do that, but I also have a habit of doing save without formatting when I'm working on files that my editor will format differently than how they're currently formatted, like the workflow yaml files. Not the case here though! Will fix.